book/templates.rst

```diff
@@ -40,12 +40,6 @@ The template of a page can be selected in the admin interface:
 
 .. figure:: ../img/templates-selection.png
 
-.. Caution::
-
-    A template is shown in the dropdown only if both the XML and the Twig file
-    exist! If you can't see your template, double-check the directories
-    ``config/templates/pages`` and ``templates/pages``.
-
```

Comment:
Should we add a info or hint for the excluded part in the webspace?

Reply:
So, what do you think about this commit 3c5641b34847a4579b117e741416923c9f9052af?